cabal-testsuite/PackageTests/NewConfigure/cabal.project.local

```diff
@@ -0,0 +1,26 @@
+verbose: verbose +nowrap +markoutput
```

Comment:
Do you really need this complicated of an initial `cabal.project.local`? Surely one or two items is enough. (There is a reason to keep it small: if any of these parameters change format / are deprecated, then you will need to update this test.)